src/course-page/CoursePageConfigCard.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { injectIntl } from '@edx/frontend-platform/i18n';
+import PropTypes from 'prop-types';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faCog } from '@fortawesome/free-solid-svg-icons';
+import { Button } from '@edx/paragon';
+import classNames from 'classnames';
+
+function CoursePageConfigCard({
+  id,
+  title,
+  description,
+  isEnabled,
+}) {
+  const cardSettings = 'bg-white shadow p-3 m-3 course-page-title';
+  const cardClasses = classNames('card', cardSettings, { disabled: !isEnabled });
+
+  const buttonSettings = 'd-flex justify-content-center align-items-center';
+  const buttonClasses = classNames(buttonSettings, { 'mt-4 visible': !isEnabled, 'mt-5 invisible': isEnabled });
+
+  return (
+    <div className={cardClasses} key={id}>
+      <div className="card-header">
+        <div className="row">
+          <h2 className="font-weight-bold">{title}</h2>
+          <div className="col flex-grow" />
+          {isEnabled && <FontAwesomeIcon icon={faCog} />}
+        </div>
+      </div>
+      <div className="card-body">
+        <p>{isEnabled ? 'Enabled' : 'Disabled'}</p>
+        <p>{description}</p>
+      </div>
+      <div className="card-footer">
+        <div className={buttonClasses}>
+          <Button className="course-page-button bg-white border font-weight-bold rounded">
+            Enable
+          </Button>
+        </div>
+      </div>
+    </div>
+  );
+}
+
+CoursePageConfigCard.propTypes = {
+  id: PropTypes.string,
+  title: PropTypes.string,
+  description: PropTypes.string,
+  isEnabled: PropTypes.bool,
+};
+
+CoursePageConfigCard.defaultProps = {
+  id: null,
+  title: null,
+  description: null,
+  isEnabled: null,
+};
+
+export default injectIntl(CoursePageConfigCard);
```

Comment:
This component is only going to be used by CoursePageResources, so move it to a subdirectory.